README.md

```diff
@@ -0,0 +1,123 @@
+      }
+    ]
+  }
+};
+```
+
+**Note:** The index page template (`index.html`), filters template partial (`filters.html`), and show page template (`show.html`) are primarily meant as a starting point for a project. They demonstrate much of the available template data, but developers will almost always want to override them to implement proper styles and layout.
+
+### Filtering by year, month, and day
+
+The default field `releaseDate` sets the date of the blog article. By default, only past articles are displayed to the user. In the Apostrophe UI, all articles are shown to the editors and admin.
+
+The blog page module, `@apostrophecms/blog-page`, provides query filters to refine blog results by year, month, and day. These are primarily used for index page filters (see the `filters.html` file), but can also be used in REST API requests and server-side queries. blogs that span multiple consecutive days are included in results if the query is at least partially included in their date span.
+
+| Filter Name | Description          | Expected Format |
+| ----------- | -------------------- | --------------- |
+| `year`      | Filter by blog year  | `YYYY`          |
+| `month`     | Filter by blog month | `YYYY-MM`       |
+| `day`       | Filter by blog day   | `YYYY-MM-DD`    |
+
+### Multiple blog piece types
+
+Sometimes a website needs multiple, distinct types of blogs. If the blog types can be managed together, it might be easiest to [add a new field](https://v3.docs.apostrophecms.org/guide/content-schema.html#using-existing-field-groups) and [query builder](https://v3.docs.apostrophecms.org/reference/module-api/module-overview.html#queries-self-query) to customize blog views. But if the blog types should be managed completely separately, it may be better to create separate piece types for each.
+
+Just as we [extend `@apostrophecms/piece-type`](https://v3.docs.apostrophecms.org/guide/pieces.html#creating-a-piece-type) to create a new piece type, we can extend `@apostrophecms/blog` to create a new blog type. The blog type will need its own module directory and UI labels. It can simply inherit the original templates, fields, and other configuration or override them in the blog type's `index.js` file.
+
+A special blog type that has an blog URL field might look like this:
+
+```javascript
+// modules/special-blog/index.js
+module.exports = {
+  extend: '@apostrophecms/blog',
+  options: {
+    label: 'Special blog',
+    pluralLabel: 'Special blogs'
+  },
+  fields: {
+    add: {
+      blogUrl: {
+        label: 'blog URL',
+        type: 'url'
+      }
+    },
+    group: {
+      basics: { fields: ['blogUrl'] }
+    }
+  }
+};
+```
+
+To display custom blog types on a page, we would need to do the same for the blog page module. All custom modules would need to be instantiated in the `app.js` file like any other module.
+
+```javascript
+// modules/special-blog-page/index.js
+module.exports = {
+  extend: '@apostrophecms/blog-page'
```

Comment:
As always with piece-page types and piece types, you must have a module extending `@apostrophecms/blog-page` that corresponds to each module extending `@apostrophecms/blog`. Apostrophe will match them up based on the naming convention.

Reply:
Done